intra_process_demo/README.md

```diff
@@ -0,0 +1,165 @@
+## **What Is This?**
+
+This demo is designed to showcase how developers can **manually compose ROS 2 nodes** by **defining them separately** but **combining them in varied process layouts**.These can be done while avoiding code overhauls or performance restrictions. 
```

Comment:
```suggestion
This demo is designed to showcase how developers can **manually compose ROS 2 nodes** by **defining them separately** but **combining them in varied process layouts**.
These can be done while avoiding code overhauls or performance restrictions. 
```

Also, I'm not quite sure what "performance restrictions" means.  Can you please elaborate?

Reply:
**performance restrictions** would refer to the **additional operations** to the following abstract:

>However, comparing the publication/reception of an intra and an inter-process message, the former requires several additional operations: it has to store the message in the ring buffer, monitor the number of Subscriptions, and extract the message.

This abstract can be found under section, **Dependent on the RMW** - **Latency and CPU utilization**. 
> https://design.ros2.org/articles/intraprocess_communications.html

Reply:
Done.